packages/node_modules/@node-red/editor-client/src/js/ui/env-var.js

```diff
@@ -0,0 +1,174 @@
+                    name: data.nameField.val(),
+                    value: data.valueField.typedInput("value"),
+                    type: data.valueField.typedInput("type")
+                };
+                new_env.push(item);
+            }
+        });
+        return new_env;
+    }
+
+    function getGlobalConf(create) {
+        var gconf = null;
+        RED.nodes.eachConfig(function (conf) {
+            if (conf.type === "global-config") {
+                gconf = conf;
+            }
+        });
+        if ((gconf === null) && create) {
+            var cred = {
+                _ : {},
+                map: {}
+            };
+            gconf = {
+                id: RED.nodes.id(),
+                type: "global-config",
+                env: [],
+                name: "global-config",
+                label: "",
+                hasUsers: false,
+                users: [],
+                credentials: cred,
+                _def: RED.nodes.getType("global-config"),
+            };
+            RED.nodes.add(gconf);
+        }
+        return gconf;
+    }
+
+    function applyChanges(list) {
+        var gconf = getGlobalConf(false);
+        var new_env = [];
+        var items = list.editableList('items');
+        var credentials = gconf ? gconf.credentials : null;
+
+        if (!credentials) {
+            credentials = {
+                _ : {},
+                map: {}
+            };
+        }
+        items.each(function (i,el) {
+            var data = el.data('data');
+            if (data.nameField && data.valueField) {
+                var item = {
+                    name: data.nameField.val(),
+                    value: data.valueField.typedInput("value"),
+                    type: data.valueField.typedInput("type")
+                };
+                if (item.name.trim() !== "") {
+                    new_env.push(item);
+                    if ((item.type === "cred") && (item.value !== "__PWRD__")) {
+                        credentials.map[item.name] = item.value;
+                        credentials.map["has_"+item.name] = (item.value !== "");
+                        item.value = "__PWRD__";
+                    }
+                }
+            }
+        });
+        if (gconf === null) {
+            gconf = getGlobalConf(true);
+        }
+        if ((JSON.stringify(new_env) !== JSON.stringify(gconf.env)) ||
+            (JSON.stringify(credentials) !== JSON.stringify(gconf.credentials))) {
+            gconf.env = new_env;
+            gconf.credentials = credentials;
+            RED.nodes.dirty(true);
+        }
+    }
+
+    function getSettingsPane() {
+        var gconf = getGlobalConf(false);
+        var env = gconf ? gconf.env : [];
+        var cred = gconf ? gconf.credentials : null;
+        if (!cred) {
+            cred = {
+                _ : {},
+                map: {}
+            };
+        }
+
+        var pane = $("<div/>", {
+            id: "red-ui-settings-tab-envvar"
```

Comment:
```suggestion
            id: "red-ui-settings-tab-envvar",
            class: "form-horizontal"
```

This fixes the padding on the env var list items to match the other env var editors.

Compare the screenshot in the PR description, with the following that shows the fixed padding:

<img width="559" alt="image" src="https://user-images.githubusercontent.com/51083/205739784-54075eb0-646b-4fc9-80c2-6ec9f49bab93.png">